src/inline.tsx

```diff
@@ -1,9 +1,12 @@
+'use client';
```

Comment:
:(
Ideally, `Inline` shouldn't need `"use client"`

Reply:
we can remove the "use client" in the next iteration and put the 1px padding unconditionally when flex-gap is not supported